src/Mod/Sketcher/Gui/EditModeCoinManager.cpp

```diff
@@ -627,6 +627,16 @@ void EditModeCoinManager::updateColor(const GeoListFacade & geolistfacade)
     pEditModeConstraintCoinManager->updateConstraintColor(constraints);
 }
 
+void EditModeCoinManager::setConstraintSelectability(bool enabled /* = true */)
+{
+    if (enabled) {

```

Comment:
I think this code should be moved to a function with the same name in EditModeConstraintCoinManager and this function should call that one. 

The reason is the SRP (single responsibility principle). EditModeConstraintCoinManager is responsible for all node management relating to constraints.

Reply:
Done